src/pages/linux/install-and-configure-apache-server-in-linux/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Install and Configure Apache Server in Linux
+---
+
+<div class="para">Apache HTTP Server, colloquially called Apache, is free and open-source 
```

Comment:
You do not need to use any kind of markup here, this is all done behind the scenes at build time